templates/s3-bucket-v2.j2

```diff
@@ -192,7 +192,7 @@ Resources:
               StringNotLike:
                 aws:userid:
                   - arn:aws:iam::325565585839:root
-                  - !Ref AccountOwnerID
+                  - !Ref AccountOwnerId
```

Comment:
Can we calle this something like AccountOwnerCanonicalId to make it clear what kind of ID it is?